hack/update-codegen.sh

```diff
@@ -19,23 +19,36 @@ set -o nounset
 set -o pipefail
 
 SCRIPT_PACKAGE=${SCRIPT_PACKAGE:-"github.com/operator-framework/operator-metering"}
-SCRIPT_ROOT="$(realpath $(dirname ${BASH_SOURCE[0]})/..)"
-CODEGEN_PKG=${CODEGEN_PKG:-$(cd ${SCRIPT_ROOT}; ls -d -1 ./vendor/k8s.io/code-generator 2>/dev/null || echo k8s.io/code-generator)}
+SCRIPT_ROOT=$(dirname ${BASH_SOURCE})/..
+CODEGEN_PKG=${CODEGEN_PKG:-$(cd ${SCRIPT_ROOT}; ls -d -1 ./vendor/k8s.io/code-generator 2>/dev/null || echo ../../../k8s.io/code-generator)}
+
+verify="${VERIFY:-}"
 
 set -x
+# Because go mod sux, we have to fake the vendor for generator in order to be able to build it...
+mv ${CODEGEN_PKG}/generate-groups.sh ${CODEGEN_PKG}/generate-groups.sh.orig
+sed 's/ go install/#go install/g' ${CODEGEN_PKG}/generate-groups.sh.orig > ${CODEGEN_PKG}/generate-groups.sh
+function cleanup {
+  mv ${CODEGEN_PKG}/generate-groups.sh.orig ${CODEGEN_PKG}/generate-groups.sh
+}
+trap cleanup EXIT
+
+go install ./${CODEGEN_PKG}/cmd/{defaulter-gen,client-gen,lister-gen,informer-gen,deepcopy-gen}
 
 # generate kubernetes client
-${CODEGEN_PKG}/generate-groups.sh "deepcopy,client,informer,lister" \
-    "${SCRIPT_PACKAGE}/pkg/generated" "${SCRIPT_PACKAGE}/pkg/apis" \
-    metering:v1 \
-    --go-header-file ${SCRIPT_ROOT}/hack/boilerplate.go.txt
+bash ${CODEGEN_PKG}/generate-groups.sh "deepcopy,client,informer,lister" \
+    "${SCRIPT_PACKAGE}/pkg/generated" \
+    "${SCRIPT_PACKAGE}/pkg/apis" \
+    "metering:v1" \
+    --go-header-file hack/boilerplate.go.txt
+    ${verify}
 
 # generate-groups doesn't do defaulters
 echo "Generating defaulters"
 ${GOPATH}/bin/defaulter-gen \
     --input-dirs "${SCRIPT_PACKAGE}/pkg/apis/metering/v1" \
     -O zz_generated.defaults \
-    --go-header-file ${SCRIPT_ROOT}/hack/boilerplate.go.txt
```

Comment:
"${SCRIPT_ROOT}/hack/boilerplate.go.txt` should still work?

Reply:
Will try for restoring all of verify-codegen.sh as-was on master. Just want to see this as-is pass e2e first.